packages/react-native/ReactAndroid/src/main/java/com/facebook/react/animated/NativeAnimatedModule.java

```diff
@@ -149,7 +152,18 @@ public long getBatchNumber() {
   }
 
   private class ConcurrentOperationQueue {
-    private final Queue<UIThreadOperation> mQueue = new ConcurrentLinkedQueue<>();
+    private final Queue<UIThreadOperation> mQueue;
+
+    public ConcurrentOperationQueue(){
+      String versionAndroid = Build.VERSION.RELEASE;
+      Log.i("VERSION12", "ConcurrentOperationQueue" + versionAndroid);
```

Comment:
I'll make this message more meaningful and log whether custom queue implementation is used or not